modules/tinymce/CHANGELOG.md

```diff
@@ -16,78 +15,80 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added a new `tablecellborderstyle` toolbar button and menu item to change table cell border style #TINY-7478
 - Added a new `tablecaption` toolbar button and menu item to toggle captions on tables #TINY-7479
 - Added a new `mceTableToggleCaption` command that toggles captions on a selected table #TINY-7479
+- Added a new `tablerowheader` toolbar button and menu item to toggle the header state of row cells #TINY-7478
+- Added a new `tablecolheader` toolbar button and menu item to toggle the header state of column cells #TINY-7482
 - Added a new `tablecellbordercolor` toolbar button and menu item to select table cell border colors, with an accompanying setting `table_border_color_map` to customize the available values #TINY-7480
 - Added a new `tablecellbackgroundcolor` toolbar button and menu item to select table cell background colors, with an accompanying setting `table_background_color_map` to customize the available values #TINY-7480
-- Added a new `initData` property to `fancymenuitem` to allow custom initialization data #TINY-7480
 - Added a new `language` menu item and toolbar button to add `lang` attributes to content, with an accompanying `content_langs` setting to specify the languages available #TINY-6149
 - A new `lang` format is now available that can be used with `editor.formatter`, or applied with the `Lang` editor command #TINY-6149
 - Added a new `language` icon for the `language` toolbar button #TINY-7670
-- Added new plugin commands: `mceEmoticons` (emoticons), `mceWordCount` (wordcount), and `mceTemplate` (template) #TINY-7619
-- Added a new `tablerowheader` toolbar button and menu item to toggle the header state of row cells #TINY-7478
-- Added a new `tablecolheader` toolbar button and menu item to toggle the header state of column cells #TINY-7482
 - Added a new `table-row-numbering` icon #TINY-7327
+- Added new plugin commands: `mceEmoticons` (Emoticons), `mceWordCount` (Word Count), and `mceTemplate` (Template) #TINY-7619
 - Added a new `iframe_aria_text` setting to set the iframe title attribute #TINY-1264
 - Added a new DomParser `Node.children()` API to return all the children of a `Node` #TINY-7756
-- Added new `FormatApply` and `FormatRemove` events #TINY-7713
 
 ### Improved
+- Sticky toolbars can now be offset from the top of the page using the new `toolbar_sticky_offset` setting #TINY-7337
+- Fancy menu items now accept an `initData` property to allow custom initialization data #TINY-7480
 - Improved the load time of the `fullpage` plugin by using the existing editor schema rather than creating a new one #TINY-6504
-- Improved the performance when rendering UI components #TINY-7572
-- When scrolling, the context toolbar will stick to where it was previously for large elements, such as tables #TINY-7545
+- Improved the performance when UI components are rendered #TINY-7572
+- When scrolling, the context toolbar will remain where it was previously for large elements, such as tables #TINY-7545
```

Comment:
```suggestion
- When scrolling, context toolbars will not reposition unneccessarily for large elements, instead remaining where they are if the position is valid
```
???? not exactly what I was thinking earlier, but along the same lines

Reply:
```suggestion
- The context toolbar will no longer unnecessarily reposition to the top of large elements when scrolling #TINY-7545
```